src/resources/postcss/tickets-admin-settings.pcss

```diff
@@ -0,0 +1,31 @@
+
+#event-tickets__admin-banner {
+	background-color: #f9f9f9;
+	border: 1px solid #ccc;
+	border-radius: 4px;
+	margin: 20px 0;
+	padding: 8px 20px 12px;
+	border-left: 5px solid #0073AA;
+
+	a {
+		text-decoration: none;
+	}
+
+	p.event-tickets__admin-banner-help-text {
+		max-width: 80%;
+	}
+
+
+	.event-tickets__admin-banner-help-links {
+		display: flex;
+
+		div {
+			min-width: 50%;
+		}
+	}
+
+	.event-tickets__admin-banner-et-links h3,
+	.event-tickets__admin-banner-etp-links h3 {
```

Comment:
Let's have a generic class and use it in both places. Let's move this class out of `event-tickets__admin-banner`